db/migrate/20160302203922_remove_is_datacenter_from_ems_folder.rb

```diff
@@ -0,0 +1,22 @@
+class RemoveIsDatacenterFromEmsFolder < ActiveRecord::Migration[5.0]
+  class EmsFolder < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+  end
+
+  def up
+    # Go through all existing EmsFolder records and set :type => Datacenter
+    # if the is_datacenter column is true before deleting it
+    EmsFolder.all.each do |folder|
+      folder.update_attributes!(:type => "Datacenter") if folder.is_datacenter?
+    end
+
+    remove_column :ems_folders, :is_datacenter
+  end
+
+  def down
+    add_column :ems_folders, :is_datacenter, :boolean
+    EmsFolder.all.each do |folder|
```

Comment:
I think you can do this in one or two update_all queries...., it's probably not a big deal but maybe...
```ruby
EmsFolder.where(:type => "Datacenter").update_all(:is_datacenter => true)
EmsFolder.where(:type => nil).update_all(:is_datacenter => false)
```